lib/export.js

```diff
@@ -126,8 +126,26 @@ module.exports = (self) => {
               return reject(error);
             }
 
+            const downloadUrl = self.apos.attachment.uploadfs.getUrl() + downloadPath;
```

Comment:
Do you really get a download this way across browsers with just `_blank`? I would think you would need to use a route that sends with with `Content-Disposition: Attachment` (I think I sent you the 2.x route we use for it).

Reply:
The only one I see that does not download is Safari, which opens the CSV as text in a new tab. This is essentially the way we handled it in the A2 version. https://github.com/apostrophecms/apostrophe-pieces-export/blob/main/public/js/export-modal.js#L39

Reply:
Since we're saving the file to the public directory I'm not sure how much we get out of putting it behind another route.